src/test/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoServiceSpec.scala

```diff
@@ -54,5 +103,74 @@ class LeonardoServiceSpec extends TestKit(ActorSystem("leonardotest")) with Flat
     }
   }
 
+  "LeonardoService" should "initialize bucket with correct files" in isolatedDbTest {
+    // set unique names for our cluster name and bucket name
+    val clusterName = s"cluster-${UUID.randomUUID.toString}"
+    val bucketName = s"bucket-${clusterName}"
+    val googleProject = s"project-${UUID.randomUUID.toString}"
+    
+    // assert that our bucket does not exist
+    assert(!gdDAO.buckets.contains(bucketName))
+
+    // create the bucket and add files
+    leo.initializeBucket(googleProject, clusterName, bucketName)
```

Comment:
Even though this returns `Future[Unit]`, might want to call `.futureValue` on it to ensure that it completes before the assertion is made on line 119.